### DISCUSSION TITLE
Training recognition script

### DISCUSSION THREAD

**Discussion by Yaswanth-165** [mindee/doctr#1331] [Q&A] [answered]
Hi I tried adding new language for training on recognition script 
When I added the language in vocab.py which is in datasets folder
So when executing the script I my new lang name as argument,  then it's throwing key error, 
So I used to print all the keys before line where I'm getting error, then I found that the changes I mage to vocab. py is not reflecting at runtime

**Comment by felixdittrich92** [comment, marked as answer]
Hi @Yaswanth-165 :wave:

You need to install it with the `-e` flag which Stands for `editable`

For example:

`cd doctr`
`pip install -e .[torch]`

**Reply by Yaswanth-165** [reply]
Thank you, it's working now
